ERCS/erc-7515.md

```diff
@@ -0,0 +1,124 @@
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+The `alt` attribute should be positioned at the root level within the metadata structure. It MUST provide a concise and accurate description of the image.
+
+```json
+{
+  "name": "NFT Name",
+  "alt": "Alternative text describing the image.",
+  "description": "Short description of the NFT image.",
+  "image": "ipfs://path_to_image",
+}
+
+```
+
+When the metadata is fetched for the NFT in the frontend, the alternative text for the image can simply be rendered as:
+
+```js
+<img src={metadata.image} alt={metadata.alt} />
+```
+
+In this context, `metadata` refers to the JSON response obtained from some fetch operation or method of retrieval. This retrieval can be executed through various means, such as using a specific library or a node provider SDK.
+
+## Rationale
+
+### 1. Guaranteed Alternative Text at Inception
+
+By embedding the alternative text during the minting phase, NFTs are equipped with accessibility features from the start. This method ensures that the foundation of accessibility is built into the NFT from the very beginning.
+
+### 2. Responsibility & Simplicity
+
+By placing the responsibility on the collection creator, we ensure that those who are most familiar with the content craft the alternative text. This not only ensures accuracy but also empowers creators to adhere to best alternative text practices—a task that, with proper guidance, is straightforward and simple.
+
+### 3. Optimal Approach
+
+This approach eliminates the potentially overwhelming task for interfaces to manually add alternative text for thousands of images and collections. The interfaces, such as marketplaces, simply need to validate that the alt field is present.
+
+### 4. Flexibility
+
+While the alt text is provided, it's not a strict imposition. Dapps retain the freedom to use the supplied alternative text, modify it, or even replace it altogether, depending on their specific needs or circumstances.
+
+## Backwards Compatibility
+
+While this ERC does not introduce a protocol-level change to existing ERC standards, there are two primary scenarios to consider:
+
+* This proposed method is not applicable to NFTs that have their metadata frozen or locked. Such NFTs will necessitate manual adjustments on the frontend to incorporate alternative text.
+
+* For NFTs that have already been minted but possess non-frozen (modifiable) metadata, updates can be made to introduce or adjust the alternative text as deemed appropriate.
+
+## Test Cases
+
+### Best Practices For Alternative Text
```

Comment:
These all should be in the Specification section, since they are giving requirements on the content of the field.